kmir/src/kmir/__main__.py

```diff
@@ -83,6 +95,7 @@ def create_argument_parser() -> ArgumentParser:
     )
     parse_subparser.add_argument(
         '--definition-dir',
+        default=None,
```

Comment:
@yanliu18 `--definition` could be omitted after this change, but it won't work unless it's a nix build. I will get it to work locally in the `feature/default-definition-paths` branch.